catalog-rest-service/src/main/resources/json/schema/entity/services/connections/database/pinotdbConnection.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/services/connections/database/pinotdbConnection.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "PinotdbConnection",
+  "description": "Pinotdb Database Connection Config",
+  "type": "object",
+  "javaType": "org.openmetadata.catalog.services.connections.database.PinotdbConnection",
+  "definitions": {
+    "pinotdbType": {
+      "description": "Service type.",
+      "type": "string",
+      "enum": ["Pinotdb"],
```

Comment:
let's follow the same naming convention that we are using for other connector,
can you please rename `Pinotdb` to `PinotDB`. (Ex: [MariaDB](https://github.com/open-metadata/OpenMetadata/blob/7f323e5ecc9670bf3fd6a1fd803abd887d066ede/catalog-rest-service/src/main/resources/json/schema/entity/services/databaseService.json#L25), [DynamoDB](https://github.com/open-metadata/OpenMetadata/blob/7f323e5ecc9670bf3fd6a1fd803abd887d066ede/catalog-rest-service/src/main/resources/json/schema/entity/services/databaseService.json#L30))